recipes/c-dbg-macro/all/conanfile.py

```diff
@@ -1,33 +1,39 @@
-from conans import ConanFile, tools
-from conans.errors import ConanInvalidConfiguration
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.files import get, copy
+from conan.tools.layout import basic_layout
 
+import os
 
-required_conan_version = ">=1.33.0"
+required_conan_version = ">=1.46.0"
 
 class DbgMacroConan(ConanFile):
     name = "c-dbg-macro"
     url = "https://github.com/conan-io/conan-center-index"
     homepage = "https://github.com/eerimoq/dbg-macro"
     license = "MIT"
     description = "A dbg(...) macro for C"
-    topics = ("conan", "debugging", "macro", "pretty-printing", "header-only")
+    topics = ("debugging", "macro", "pretty-printing", "header-only")
     settings = ("compiler",  "build_type", "os", "arch")
     no_copy_source = True
 
-    @property
-    def _source_subfolder(self):
-        return "source_subfolder"
-
     def validate(self):
-        if self.settings.os == "Windows":
+        if self.info.settings.os == "Windows":
```

Comment:
The info with header only does not work in 1,x 🙈 this will work in with develop2 branch :)

```suggestion
        if self.settings.os == "Windows":
```